.travis.yml

```diff
@@ -22,6 +22,7 @@ env:
         - ES_VERSION=5.6.10
         - ES_DOWNLOAD_URL=https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.6.10.tar.gz
         - PIPELINE_CLEANCSS_BINARY=$TRAVIS_BUILD_DIR/node_modules/.bin/cleancss
+        - PIPELINE_CLEANCSS_ARGUMENTS="-O2 --skip-rebase"
```

Comment:
I don't know what uses travis.yml, so I don't know how to test that this works. In particular I don't know whether the double quotes here are the right thing.

Reply:
Well, the travis ran automatically when I created the PR, so presumably that means that this worked.